### PR TITLE
Tests the instantiation of most movables

### PR DESCRIPTION
**Pull request by Cyberboss**
Whoo boy I like pain

### CODE REVIEW

**Comment by Cyberboss** [comment]
Do not stale this PR, gonna be addressing each issue seperately

File: code/modules/unit_tests/movable_sanity.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    
+    for(var/I in typesof(/mob))
+        if(blacklist[I])
+            continue
+        ProcessType(I)
+
+/datum/unit_test/movable_sanity/proc/ProcessType(target_type)
+    var/list/target_arguments
+    var/list/_arguments = arguments
+    for(var/current_type = target_type; !target_arguments && current_type != /atom/movable; current_type = type2parent(current_type))
```
Comment (Iamgoofball) [inline code/modules/unit_tests/movable_sanity.dm]: >semicolons

Comment (Cyberboss) [reply code/modules/unit_tests/movable_sanity.dm]: It's normal for loop syntax, very valid for what's going on here

**Review by coiax — COMMENTED** [review]
Where will all these things be existed into being, and will you be able to watch?

**Comment by Cyberboss** [comment]
It's just a random patch of space. And no, unless you run the test locally
I suppose...

**Comment by JJRcop** [comment]
@coiax This is supposed to happen on one of the travis builds, and won't normally happen in game unless you compile with UNIT_TESTS defined.

**Comment by Cyberboss** [comment]
Bigger better things, on the todo list it goes.

**Comment by JJRcop** [comment]
Could you push any new changes that may be on your local repo but not github?

**Comment by Cyberboss** [comment]
It's all there

**Pull request closed without merging**